go-controller/pkg/clustermanager/obj_retry_cm.go

```diff
@@ -0,0 +1,213 @@
+package clustermanager
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/factory"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/retry"
+	kapi "k8s.io/api/core/v1"
+	cache "k8s.io/client-go/tools/cache"
+	"k8s.io/klog/v2"
+)
+
+type cmEventHandler struct {
+	retry.EventHandler
+
+	objType         reflect.Type
+	cm              *ClusterManager
+	extraParameters interface{}
```

Comment:
since we don't use it, we can remove `extraParameters` from the struct

Reply:
Ack